connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -1273,7 +1272,20 @@ export async function firecrawlCrawlCompleted(
     return;
   }
 
-  await syncSucceeded(connector.id);
+  const crawlStatus = await getFirecrawl().checkCrawlStatus(crawlId);
+  if (!crawlStatus.success) {
+    localLogger.error(
+      { connectorId, crawlId },
+      `Couldn't fetch crawl status: ${crawlStatus.error}`
+    );
+    return;
+  }
+
+  if (crawlStatus.completed >= crawlStatus.total) {
```

Comment:
Other way around no?

Reply:
No, `completed` is the equivalent of the previous `requestFinished`, per doc: "The number of pages that have been successfully crawled."
and `total` is the "The total number of pages that were attempted to be crawled.", but saying that, that would be not the same behavior as currently, as it can be different than the `maxPageToCrawl`, updating the PR.